src/queues.c

```diff
@@ -299,18 +299,20 @@ static HSAKMT_STATUS map_doorbell(HSAuint32 NodeId, HSAuint32 gpu_id,
 static void *allocate_exec_aligned_memory_cpu(uint32_t size, uint32_t align)
```

Comment:
If we're always using this with PAGE_SIZE, and PAGE_SIZE is the only supported alignment, we may as well remove the align parameter.